client/lib/performance/index.js

```diff
@@ -0,0 +1,7 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import measure from './measured-component';
+
+export { measure };
```

Comment:
```js
export { measure } from './measured-component';
```

???

also, why does this file even exist? why not import from `client/lib/performance/measured-component` directly?

Reply:
good Q! My thought is that it would be nice to add more organization to our perf tooling. we already have something called `perfmon` as well that I could see moved here.